deploy/helm-chart/harbor-operator/crds/registries.mittwald.de_repository_crd.yaml

```diff
@@ -39,45 +39,24 @@ spec:
           type: object
         spec:
           properties:
-            CVEWhitelist:
-              properties:
-                id:
-                  format: int64
-                  type: integer
-                items:
-                  items:
-                    properties:
-                      CVEID:
-                        type: string
-                    type: object
-                  type: array
-                projectID:
-                  format: int64
-                  type: integer
-              required:
-              - id
-              - projectID
-              type: object
-            deleted:
-              type: boolean
             memberRequests:
               description: Ref to the name of a 'User' resource
               items:
                 properties:
-                  memberUser:
+                  role:
+                    type: string
+                  user:
+                    description: LocalObjectReference contains enough information
+                      to let you locate the referenced object inside the same namespace.
```

Comment:
These files are changed, because the source files were changed in a previous PR, I assume? Maybe for the future we should add a workflow (in a separate PR, not here) that asserts that all generated code is actually up-to-date for each PR?